test/system/700-play.bats

```diff
@@ -981,3 +981,45 @@ _EOF
     run_podman pod rm -t 0 -f test_pod
     run_podman rmi -f userimage:latest $from_image
 }
+
+@test "podman play with automount volume" {
+    cat >$PODMAN_TMPDIR/Containerfile <<EOF
+FROM $IMAGE
+RUN sh -c 'mkdir /test1 /test2'
+RUN sh -c 'touch /test1/a /test1/b /test1/c'
+RUN sh -c 'touch /test2/asdf /test2/ejgre /test2/lteghe'
+VOLUME /test1
+VOLUME /test2
+EOF
+
+    run_podman build -t automount_test -f $PODMAN_TMPDIR/Containerfile
+
+    fname="/tmp/play_kube_wait_$(random_string 6).yaml"
+    echo "
+apiVersion: v1
+kind: Pod
+metadata:
+  labels:
+    app: test
+  name: test_pod
+spec:
+  restartPolicy: Never
+  containers:
+    - name: test1
+      image: $IMAGE
+      command:
+      - top
+" > $fname
+
+    run_podman kube play --annotation "io.podman.annotations.kube.image.volumes.mount/test1=automount_test" $fname
+
+    run_podman run --rm automount_test ls /test1
+    run_out_test1="$output"
+    run_podman exec test_pod-test1 ls /test1
+    assert "$output" = "$run_out_test1"
+
+    run_podman run --rm automount_test ls /test2
+    run_out_test2="$output"
+    run_podman exec test_pod-test1 ls /test2
+    assert "$output" = "$run_out_test2"
```

Comment:
...and likewise here, "exec ls /test2".

That makes it possible for a human viewing test results to understand what happened without having to scroll up, look at line numbers, scroll back down, etc